src/NHSD.BuyingCatalogue.Solutions.Application/Queries/GetRoadMapBySolutionId/GetRoadMapByIdHandler.cs

```diff
@@ -18,10 +18,12 @@ public GetRoadMapByIdHandler(RoadMapReader roadMapReader, SolutionVerifier verif
             _verifier = verifier;
         }
 
-        public async Task<string> Handle(GetRoadMapByIdQuery request, CancellationToken cancellationToken)
+        public async Task<IRoadMap> Handle(GetRoadMapBySolutionIdQuery request, CancellationToken cancellationToken)
         {
             await _verifier.ThrowWhenMissing(request.Id, cancellationToken).ConfigureAwait(false);
-            return (await _roadMapReader.ByIdAsync(request.Id, cancellationToken).ConfigureAwait(false)).Description;
+            var roadMapResult = (await _roadMapReader.BySolutionIdAsync(request.Id, cancellationToken).ConfigureAwait(false));
+            return new RoadMapDto { Summary = roadMapResult?.Summary };
```

Comment:
Could we use the mapper here? Follow the convention of the rest of the handlers.